import-export-cli/impl/exportAPI.go

```diff
@@ -63,6 +63,8 @@ func exportAPI(name, version, revisionNum, provider, format, publisherEndpoint,
 	headers[utils.HeaderAuthorization] = utils.HeaderValueAuthBearerPrefix + " " + accessToken
 	headers[utils.HeaderAccept] = utils.HeaderValueApplicationZip
 
+	fmt.Println("API export endpoint: ", url)
```

Comment:
Do we need this?

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)